mapbox/services/uploads.py

```diff
@@ -54,8 +55,13 @@ def _validate_tileset(self, tileset):
         """
         if '.' not in tileset:
             tileset = "{0}.{1}".format(self.username, tileset)
-        if len(tileset) > 64:
-            raise ValidationError('tileset including username must be < 64 char')
+
+        pattern = '^[a-z0-9-_]{1,32}\.[a-z0-9-_]{1,32}$'
```

Comment:
@perrygeo part of me says this could be a module level constant for more visibility, but part of me says don't bother. What do you think?

Reply:
lets wait until we have a use case for exposing the pattern.